pkg/metricscollector/v1beta1/tfevent-metricscollector/tfevent_loader.py

```diff
@@ -72,7 +78,19 @@ def parse_summary(self, tfefile):
                         ),
                     )
                     metric_logs.append(ml)
-
+        # support old-style tensorboard metrics too
+        for tag in tags[SCALARS]:
+            for m in self.metric_names:
+                if not _should_consider(tag, m, tfefile):
```

Comment:
thanks @Electronic-Waste for trusting me on this. 
@garymm I'd suggest some changes here:

how about:

```python
for tag in tags[SCALARS]:
        for metric_name in self.metric_names:
            if _should_consider(tag, metric_name, tfefile):
                for scalar in event_accumulator.Scalars(tag):
                    metric_logs.append(
                        api_pb2.MetricLog(
                            time_stamp=rfc3339.rfc3339(datetime.fromtimestamp(scalar.wall_time)),
                            metric=api_pb2.Metric(name=metric_name, value=str(scalar.value)),
                        )
                    )

    return metric_logs
```